conans/test/command/package_test.py

```diff
@@ -15,30 +15,43 @@ def package_with_destination_test(self):
         client = TestClient()
 
         def prepare_for_package(the_client):
+            the_client.all_output = ""
             the_client.save({"src/header.h": "contents"}, clean_first=True)
             the_client.run("new lib/1.0 -s")
 
-            # don't need real build
+            # don't need build method
             tools.replace_in_file(os.path.join(client.current_folder,
-                                               "conanfile.py"), "cmake =",
-                                  "return\n#")
+                                               "conanfile.py"),
+                                               "def build(self):",
+                                               "#def build(self):")
+            tools.replace_in_file(os.path.join(client.current_folder,
+                                               "conanfile.py"),
+                                               "cmake =",
+                                               "#cmake =")
+            tools.replace_in_file(os.path.join(client.current_folder,
+                                               "conanfile.py"),
+                                               "cmake.",
+                                               "#cmake.")
             the_client.run("install . --install-folder build")
-            the_client.run("build . --build-folder build2 --install-folder build")
+            mkdir(os.path.join(client.current_folder, "build2"))
 
         # In current dir subdir
         prepare_for_package(client)
         client.run("package . --build-folder build2 --install-folder build --package_folder=subdir")
+        self.assertTrue("package(): WARN: No files copied!" not in client.all_output)
```

Comment:
use ``self.assertNotIn``
Also, check in ``client.out``, which is only last command output, not ``all_output`` that accumulates